### DISCUSSION TITLE
Dataset with all environmental impact categories

### DISCUSSION THREAD

**Discussion by tilseth** [USEPA/supply-chain-factors#77] [Q&A] [answered]
How is it possible to create a dataset that contains not only the GHG EFs for each NAICS code, but also includes all other environmental impact categories (acidification, eutrophication, etc.)?

**Comment by bl-young** [comment, marked as answer]
Good question - adding additional satellite data and indicators is as easy as expanding the [model spec file](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml).

For examples of what you can include here and published datasets you can check out the published [USEEIOv2.0.1-411 model](https://github.com/USEPA/useeior/blob/v1.5.1/inst/extdata/modelspecs/USEEIOv2.0.1-411.yml), which has a much greater set of satellite data and indicators (see the [publication](https://www.nature.com/articles/s41597-022-01293-7)).

Generating factors with an updated model with more data will likely require some minor adjustments to [CalculateEmissionFactors.Rmd](https://github.com/USEPA/supply-chain-factors/blob/main/CalculateEmissionFactors.Rmd) and the supporting [Functions.R](https://github.com/USEPA/supply-chain-factors/blob/main/R/Functions.R) within this package to be more flexible with the indicator(s) you choose, but should be feasible with a little bit of R experience.
